client/model/toolsModel.js

```diff
@@ -0,0 +1,110 @@
+define( [
+        'knockout',
+        'service',
+        'jquery',
+        'model/pillarApiModel',
+        'model/saltModel',
+        'bindings/uppercase'
+    ],
+    function(ko, service, $) {
+        return function toolsModel(login) {
+            var self = this;
+            self.login = login;
+            self.oldPath = ko.observable('');
+            self.newPath = ko.observable('/spot/software/state/application/');
```

Comment:
I would use the value from constants.js (line 62)
this way if we ever have to change it we only have to do it one place